packages/nextra-theme-docs/src/components/toc.tsx

```diff
@@ -57,12 +57,18 @@ export function TOC({ toc, filePath }: TOCProps): ReactElement {
   return (
     <div
       className={cn(
-        'nextra-scrollbar _sticky _top-16 _overflow-y-auto _px-4 _pt-6 _text-sm [hyphens:auto]',
+        '_grid _grid-rows-[1fr_min-content]', // 1fr: scrollbar, min-content: toc footer
+        '_sticky _top-16 _pt-6 _text-sm [hyphens:auto]',
```

Comment:
```suggestion
        '_sticky _top-[--nextra-navbar-height] _pt-6 _text-sm [hyphens:auto]',
```
let's replace this magic number, as I did in Nextra 4 https://github.com/shuding/nextra/blob/b476b38ddf0ec2c09aaa40a8cf3de37aaa029754/packages/nextra-theme-docs/src/components/toc.tsx#L59C35-L59C64